### ISSUE TITLE
 DOMException: Blocked a frame with origin "lsp://logseq.io"

### ISSUE THREAD

**Issue by Bad3r** [pengx17/logseq-plugin-todo-master#5]
Hi,

I noticed an error in logseq console related to the plugin  
```
[LSPlugin Ready Error] DOMException: Blocked a frame with origin "lsp://logseq.io" from accessing a cross-origin frame.
    at ke.P (lsp://logseq.io/logseq-todo-master/dist/assets/index.87bfa7fe.js:85:3012)
    at ke.ready (lsp://logseq.io/logseq-todo-master/dist/assets/vendor.c775ebdb.js:10:17058)

```

**Comment by pengx17** [comment]
This is strange. Is this issue introduced in version 0.5.3? Let me take a look.

**Comment by pengx17** [comment]
Oh, thanks! I forget to remove some old code. The old implementation relies on reading the `top` context (main window), but it is no longer required.

**Comment by pengx17** [comment]
fixed by cc26bc2

**Comment by Bad3r** [comment]
Thank you! :)
